ai-platform-unified/notebooks/official/automl/tabular-model-training-and-prediction.ipynb

```diff
@@ -0,0 +1,733 @@
+{
```

Comment:
<p>You could just do:</p><p> </p><pre class="ql-syntax" spellcheck="false">endpoint = model.deploy(
&nbsp; &nbsp; machine_type="n1-standard-4",
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>